http/swagger.yml

```diff
@@ -4341,6 +4341,11 @@ paths:
           schema:
             type: string
           description: Filter tasks to a specific organization ID.
+        - in: query
+          name: active
+          schema:
+            type: string
+          description: Filter tasks to a specific status--a value of "true" corresponds to active, "false" to inactive.
```

Comment:
A downside to using a boolean here is that it would be harder to extend the API if we were to add a third type of status for tasks in the future.

Reply:
Agreed, seems `status` as the name and `active` and `inactive` as current acceptable contents would be a better fit.

Reply:
Originally we weren't going to expose `active` via the HTTP API, but clearly a valid request a client would wish to make.  @AlirieGray and I had a discussion and for parity, will expose `status` as a string enum. 👍